### PR TITLE
Auto/separate readme behavior velocity

### PR DESCRIPTION
**Pull request by taikitanaka3**
## Description

separate readme

## Review Procedure

see docs

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/README.md [hunk 1]
```diff
@@ -53,714 +54,3 @@ So for example, in order to stop at a stop line with the vehicles' front on the
 | `delay_response_time`   | double | (to be a global parameter) delay time of the vehicle's response to control commands |
 
 ## Modules
```
Comment (TakaHoribe) [inline planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/README.md:56]:
```suggestion

```

File: planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/README.md [hunk 2]
```diff
@@ -5,13 +5,14 @@
 `behavior_velocity_planner` is a planner that adjust velocity based on the traffic rules.
 It consists of several modules.
```
Comment (TakaHoribe) [inline planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/README.md:6]:
```suggestion
It consists of several modules. Please refer to the links listed below for detail on each module.
```

File: planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection-design.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+To prevent a chattering, once the ego vehicle passes this line, “stop” decision in the intersection won’t be done any more.
+To prevent going over the pass judge line before the traffic light stop line, the distance between stopline and pass judge line become 0m in case that there is a stop line between the ego vehicle and an intersection stop line.
+
+#### Vehicle In a Same Lane Removal
+
+Ignore the object in front of/ behind the ego vehicle in the same lane (Improvement needed : the linkage with the uuid in tracking is needed)
+
+#### Stuck vehicle
+
+If there is any object in a certain distance (default : 5m) from the end point of the intersection lane on the driving lane and a infered velocity of the object is less than a threshold (default 3.0km/h), the object is regarded as a stuck vehicle. If the stuck vehicle exists, the ego vehicle cannot enter the intersection.
```
Comment (TakaHoribe) [inline planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection-design.md:50]:
```suggestion
If there is any object in a certain distance (default : 5m) from the end point of the intersection lane on the driving lane and the object velocity is less than a threshold (default 3.0km/h), the object is regarded as a stuck vehicle. If the stuck vehicle exists, the ego vehicle cannot enter the intersection.
```

File: planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection-design.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+| `intersection/intersection_max_accel`         | double | [m/s^2] acceleration in intersection                                          |
+| `intersection/detection_area_margin`          | double | [m] range for expanding detection area                                        |
+| `intersection/detection_area_length`          | double | [m] range for lidar detection 200m is by default                              |
+| `intersection/detection_area_angle_threshold` | double | [rad] threshold of angle difference between the detection object and lane     |
+| `intersection/min_predicted_path_confidence`  | double | [-] minimum confidence value of predicted path to use for collision detection |
+| `merge_from_private_road/stop_duration_sec`   | double | [s] duration to stop                                                          |
+
+### Usage And Parameter Tuning
+
+Incase
```
Comment (TakaHoribe) [inline planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection-design.md:88]: これどういう意味？

File: planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection-design.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+| `intersection/stuck_vehicle_vel_thr`          | double | [m/s] velocity below 3[km/h] is ignored by default                            |
+| `intersection/intersection_velocity`          | double | [m/s] velocity to pass intersection. 10[km/h] is by default                   |
+| `intersection/intersection_max_accel`         | double | [m/s^2] acceleration in intersection                                          |
+| `intersection/detection_area_margin`          | double | [m] range for expanding detection area                                        |
+| `intersection/detection_area_length`          | double | [m] range for lidar detection 200m is by default                              |
+| `intersection/detection_area_angle_threshold` | double | [rad] threshold of angle difference between the detection object and lane     |
+| `intersection/min_predicted_path_confidence`  | double | [-] minimum confidence value of predicted path to use for collision detection |
+| `merge_from_private_road/stop_duration_sec`   | double | [s] duration to stop                                                          |
+
+### Usage And Parameter Tuning
```
Comment (taikitanaka3) [inline planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection-design.md:86]:
```suggestion
### How To Tune Parameters
```

File: planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection-design.md (hunk 4 above)
Comment (taikitanaka3) [inline planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection-design.md:88]:
```suggestion

```

Comment (taikitanaka3) [reply planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection-design.md]: 直接話して削除することに

**Review by TakaHoribe — APPROVED** [review]

**Pull request merged**